apps/agent/package.json

```diff
@@ -36,12 +36,10 @@
     "@aragon/test-helpers": "^1.1.0",
     "eth-gas-reporter": "^0.1.1",
     "ethereumjs-util": "^6.1.0",
-    "solidity-coverage": "0.5.11",
     "solium": "^1.2.3",
     "truffle": "4.1.14",
     "truffle-extract": "^1.2.1",
-    "web3-eth-abi": "^1.0.0-beta.38",
-    "webpack": "3.10.0"
```

Comment:
We used to have to pin webpack because the frontends relied on it and somehow there were version clashes with ganache (or something weird), but we've since moved to parcel and nothing should rely on webpack now.